src/siotls/examples/simple_client.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from socket import socket
+from siotls import TLSConfiguration, TLSConnection
+from siotls.utils import hexdump
+
+
+logger = logging.getLogger(__name__)
+
+def connect(host, port):
+    server = socket()
```

Comment:
would name it this way
```suggestion
    client = socket()
```